src/lib/controller/store.test.ts

```diff
@@ -31,27 +39,36 @@ describe("StoreController", () => {
     completions: 1,
   };
 
+  beforeEach(() => {
+    jest.clearAllMocks(); // Clear all mocks before each test
+    mockLog.info.mockClear(); // Explicitly clear the logger mock
+    testCapability = new Capability(capabilityConfig);
+    const mockImplementation = createMockImplementation();
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
We generally prefer to avoid the use of any, but in this case properly mocking this object requires a lot of scaffolding that is not worth the effort for the level of confidence these tests provide.